### PR TITLE
Added script to build all the configurations

### PR DESCRIPTION
**Pull request by lulivi**

### CODE REVIEW

**Comment by lulivi** [comment]
Should I provide the same copyright notice as I provided in the aws python script?

**Comment by iblancasa** [comment]
@lulivi yes

File: resources/cmake/ConnextDDSBuildAll.cmake [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+        file(MAKE_DIRECTORY "${BINARY_DIR_PATH}")
+
+        execute_process(
+            COMMAND ${CMAKE_COMMAND}
+                -DCMAKE_BUILD_TYPE=${build_type}
+                -DBUILD_SHARED_LIBS=${link_type}
+                ${CMAKE_CURRENT_SOURCE_DIR}
+            WORKING_DIRECTORY "${BINARY_DIR_PATH}")
+        execute_process(
```
Comment (iblancasa) [inline resources/cmake/ConnextDDSBuildAll.cmake]: I think you can add more than one `COMMAND` per `execute_process` call

Comment (lulivi) [reply resources/cmake/ConnextDDSBuildAll.cmake]: If I join both processes in one like this:

```cmake
execute_process(
    COMMAND ${CMAKE_COMMAND}
        -DCMAKE_BUILD_TYPE=${build_type}
        -DBUILD_SHARED_LIBS=${link_type}
        ${CMAKE_CURRENT_SOURCE_DIR}
    COMMAND ${CMAKE_COMMAND}
        --build .
        --config ${build_type}
    WORKING_DIRECTORY "${BINARY_DIR_PATH}")
```

It throws an error:

```
-- Building Debug with Static linking...
Error: could not load cache
-- Building Debug with Dynamic linking...
Error: could not load cache
-- Building Release with Static linking...
Error: could not load cache
-- Building Release with Dynamic linking...
Error: could not load cache
```

**Pull request merged**